src/components/DiscordWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import "./DiscordWidget.css";
+
+export default function DiscordWidget() {
+    const [info, setInfo] = useState({ online: "???", members: "???" });
+    useEffect(() => {
+        const async = async () => {
```

Comment:
Some better name for this?

Reply:
Better name than DiscordWidget ?  
It's a good name IMO.  
It's the same thing than the discord widget but with reduced functionality.

Reply:
The `const async` not the widget